packages/landing/src/app/common/space.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components'
+
+const Space = styled.div<{
+  height?: number
+  heightMobile?: number
+  width?: number
+  widthMobile?: number
+  xl?: boolean
+  md?: boolean
+  sm?: boolean
+  xs?: boolean
```

Comment:
Let's left only `height` and `heightMobile` properties that we actually use.

Reply:
Ok, done it